install.sh

```diff
@@ -1019,9 +1019,9 @@ then
   then
     plural="s"
   fi
-  printf "- Run these commands in your terminal to add the non-default Git remote%s for %s:" "${plural}" "${non_default_repos}"
-  printf "    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> %s" "${shell_profile}"
-  printf "    echo '%s' >> %s\n" "${additional_shellenv_commands[@]}" "${shell_profile}"
+  printf -- "- Run these commands in your terminal to add the non-default Git remote%s for %s:\n" "${plural}" "${non_default_repos}"
+  printf "    echo '# Set PATH, MANPATH, etc., for Homebrew.' >> %s\n" "${shell_profile}"
+  printf "    echo '%s' >> ${shell_profile}\n" "${additional_shellenv_commands[@]}"
```

Comment:
Can you explain this change? I'm not sure I understand it.

Reply:
Addition of `\n` on line 1023 is 👍🏻.
Change on line 1024 seems to be unnecessary.

Reply:
> Can you explain this change? I'm not sure I understand it.

When multiple environment variables are set, it looks like this:

<img width="1167" alt="snap" src="https://user-images.githubusercontent.com/18101258/202228228-7d27ba24-0429-4de1-a382-1cee1e8aeca8.png">

Reply:
Ok, thanks.